src/manifest.ts

```diff
@@ -715,6 +721,9 @@ export class Manifest {
             path,
             pullRequest,
             draft: config.draft ?? this.draft,
+            prerelease:
+              (config.prerelease ?? this.prerelease) &&
```

Comment:
same as above, we shouldn't need to have `this.prerelease`

Reply:
I agree. Would the same apply to line 723?

Reply:
Yes, we should clean that up, but maybe in a separate cleanup PR?

Reply:
I looked into the case with having `draft` and `draftPullRequest` on `Manifest`. my conclusion is the order should be `this.draft ?? config.draft` since some `cli` commands can take `--draft` and `--draft-pull-request` in addition to a manifest file.  Eg. here https://github.com/googleapis/release-please/blob/580f6290e89da56f4e5afc97d8c25fdccbabca7a/src/bin/release-please.ts#L424-L430

I guess then the flag should override the behaviour from the config file (at least what I infered from the tests). I created a pr on my fork to have a better look on my mods there https://github.com/kalosisz/release-please/pull/7

Did I understend correctly the role of the two flags?

Reply:
I updated the logic to only consider the `config`